neofs_config.mk

```
@@ -1,26 +1,31 @@
 # Update epoch duration in side chain blocks (make update.epoch_duration val=30)
 update.epoch_duration:
-	@./bin/config.sh EpochDuration $(val)
+	@./bin/config.sh int EpochDuration $(val)
 
 # Update max object size in bytes (make update.max_object_size val=1000)
 update.max_object_size:
-	@./bin/config.sh MaxObjectSize $(val)
+	@./bin/config.sh int MaxObjectSize $(val)
 
 # Update audit fee per result in fixed 12 (make update.audit_fee val=100)
 update.audit_fee:
-	@./bin/config.sh AuditFee $(val)
+	@./bin/config.sh int AuditFee $(val)
 
 # Update container fee per alphabet node in fixed 12 (make update.container_fee val=500)
 update.container_fee:
-	@./bin/config.sh ContainerFee $(val)
+	@./bin/config.sh int ContainerFee $(val)
 
 # Update amount of EigenTrust iterations (make update.eigen_trust_iterations val=2)
 update.eigen_trust_iterations:
-	@./bin/config.sh EigenTrustIterations $(val)
+	@./bin/config.sh int EigenTrustIterations $(val)
+
+
+# Update alpha parameter of EigenTrust algorithm in 0 <= f <= 1.0 (make update.eigen_trust_value val=0.2)
+update.eigen_trust_value:
+	@./bin/config.sh string EigenTrustValue $(val)
```

i guess it is a mistake in the issue
```suggestion
	@./bin/config.sh string EigenTrustAlpha $(val)
```

@alexvanin?

---

Yes, my bad, should be `EigenTrustAlpha`

---

Fixed